src/components/Icon/Icon.tsx

```diff
@@ -31,3 +31,9 @@ export const Icon = ({
         />
     );
 };
+
+// When used with uikit components Icon is considered as text element and corresponding styles are applied
+// IconWrapper overrides displayName to 'Icon', so it will be considered as an icon with right styles
+export const IconWrapper = (props: IconProps) => <Icon {...props} />;
+
+IconWrapper.displayName = 'Icon';
```

Comment:
I realised that we even do not need Wrapper.
We may just do `Icon.displayName = 'Icon'` and that's it!

Reply:
Ok, lets do an issue and refactor it in the future!

Reply:
Added issue - https://github.com/ydb-platform/ydb-embedded-ui/issues/344